sources/bloodhound/src/bin/kubernetes-checks/checks.rs

```diff
@@ -649,3 +649,66 @@ impl Checker for K8S04021200Checker {
         }
     }
 }
+
+// =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<=
+
+pub struct K8S04021300Checker {}
+
+impl Checker for K8S04021300Checker {
+    fn execute(&self) -> CheckerResult {
+        let allowed_suites = vec![
+            "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+            "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+            "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
+            "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+            "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+            "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+            "TLS_RSA_WITH_AES_256_GCM_SHA384",
+            "TLS_RSA_WITH_AES_128_GCM_SHA256",
+        ];
+
+        #[derive(Deserialize)]
+        struct KubeletConfig {
+            #[serde(rename = "tlsCipherSuites")]
+            tls_cipher_suites: Vec<String>,
+        }
+
+        let mut result = CheckerResult::default();
+
+        if let Ok(kubelet_file) = File::open(KUBELET_CONF_FILE) {
+            if let Ok(config) = serde_yaml::from_reader::<_, KubeletConfig>(kubelet_file) {
+                let mut allowed = true;
+                for suite in config.tls_cipher_suites {
+                    if !allowed_suites.iter().any(|&i| i == suite) {
+                        allowed = false;
+                        result.error = format!("'{}' not an allowed cipher suite", suite);
+                        break;
+                    }
+                }
```

Comment:
the lists are small, but this would avoid the O(nm) algorithm:
```
let configured_suites = HashSet::from(config.tls_cipher_suites);
if !configured_suites.is_subset(allowed_suites) {
    result.error = format!("Found disallowed cipher suites");
}
```